transports/postgresql/transactions.md

```diff
@@ -12,10 +12,10 @@ The PostgreSQL transport supports the following [transport transaction modes](/t
 * Transport transaction - receive only
 * Unreliable (transactions disabled)
 
-`TransactionScope` mode is particularly useful as it enables `exactly once` message processing with distributed transactions. However, when transport, persistence, and business data are all stored in a single PostgreSQL catalog, it is possible to achieve `exactly-once` message delivery without distributed transactions.
+`TransactionScope` mode is not supported because the implementation of `TransactionScope` in [npgsql](https://www.npgsql.org/index.html) (ADO.NET driver for PostgreSQL) is not reliable and may lead to logical message loss (an update to a database that is prepared and persisted but not actually applied to the data tables).
```

Comment:
is there an external link available that explains the message loss scenario?

Reply:
Unfortunately no and the npgsql maintainers didn't acknowledge it. Here's the issue we raised https://github.com/npgsql/npgsql/issues/5683

Reply:
```suggestion
`TransactionScope` mode is not supported because the implementation of `TransactionScope` in [npgsql](https://www.npgsql.org/index.html) (ADO.NET driver for PostgreSQL) is not reliable and [may lead to logical message loss](https://github.com/npgsql/npgsql/issues/5683) (an update to a database that is prepared and persisted but not actually applied to the data tables).
```